pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiCompositeAnswer.java

```diff
@@ -89,7 +90,8 @@
                                                 rs.getString("child_answer_guid"),
                                                 rs.getLong("child_question_id"),
                                                 rs.getString("child_question_stable_id"),
-                                                QuestionType.valueOf(rs.getString("child_question_type_code"))));
+                                                QuestionType.valueOf(rs.getString("child_question_type_code")),
+                                                rs.getLong("parent_activity_instance_id")));
```

Comment:
It should be safe to assume that child answers have the same `activity_instance_id` as the parent. CC @MocanaAtBroad